src/App.tsx

```diff
@@ -10,33 +12,29 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [tabId, setTabId] = useState(tabs[0].id);
+  const [title, setTitle] = useState(tabs[0].title);
```

Comment:
You don't need a separate state for each option. Just create a general one. You can find a title if you know the id already. Refactor your solution according to this comment, please 
```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```